ql/ABQLSetPluckCore.js

```diff
@@ -30,7 +30,7 @@ class ABQLSetPluckCore extends ABQL {
       // #Hack! : when an Operation provides the same .NextQlOps that it
       // was defined in, we can't require it again ==> circular dependency.
       // so we manually set it here from the operation that created us:
-      this.constructor.NextQLOps = this.prevOP.constructor.NextQLOps;
+      this.constructor.NextQLOps = (this.constructor.NextQLOps == null || this.constructor.NextQLOps.length < 1 ? this.prevOP.constructor.NextQLOps : this.constructor.NextQLOps);
```

Comment:
the ```this.prevOP.constructor.NextQLOps``` might be a good way to handle the ```*Save``` operations passing on the same operations in the Chain.